labelme/widgets/canvas.py

```diff
@@ -403,13 +414,12 @@ def mouseReleaseEvent(self, ev):
                 # Cancel the move by deleting the shadow copy.
                 self.selectedShapesCopy = []
                 self.repaint()
-        elif ev.button() == QtCore.Qt.LeftButton and self.selectedVertex():
-            if (
-                self.editing()
-                and int(ev.modifiers()) == QtCore.Qt.ShiftModifier
-            ):
-                # Delete point if: left-click + SHIFT on a point
-                self.removeSelectedPoint()
+        elif ev.button() == QtCore.Qt.LeftButton:
+            if self.editing():
+                if self.wasSelected and not self.movingShape:
```

Comment:
`self.wasSelected` can be replaced with `self.hShape in self.selectedShapes`?

Reply:
Hmm ... I think no because `self.hShape` is always in `self.selectedShapes` when it's called at `mouseReleaseEvent()` after the `selectShapePoint()`. That's what `self.wasSelected` do ... Makes a difference between newly added shape and previously selected shape (both are in `self.selectedShapes` until Line 425).